### DISCUSSION TITLE
Amber API pricing

### DISCUSSION THREAD

**Discussion by tux43** [hif2k1/battery_sim#124] [Ideas]
Is it possible to utilise the Amber API for the wholesale price tariff rate. This will help those considering Amber and a battery to determine a payback period.

https://app.amber.com.au/developers/documentation/

**Comment by hif2k1** [comment]
I think if you use this integration you should be able to:
https://www.home-assistant.io/integrations/amberelectric/

**Comment by dewi-ny-je** [comment]
This project is about a battery simulator. Defining the prices is literally not a battery related thing. The simulator offers configuration parameters for prices, that's as far as it should go.
